packages/cloudflare/src/cli/build/patches/to-investigate/patch-exception-bubbling.ts

```diff
@@ -5,5 +5,14 @@
  * promises.
  */
 export function patchExceptionBubbling(code: string) {
-  return code.replace('_nextBubbleNoFallback = "1"', "_nextBubbleNoFallback = undefined");
+  // The code before had: `query._nextBubbleNoFallback = '1'`, that has ben refactored to
+  // `addRequestMeta(req, 'bubbleNoFallback', true)` in https://github.com/vercel/next.js/pull/74100
+  // we need to support both for backward compatibility, that's why we have the following if statement
+  if (code.includes("_nextBubbleNoFallback")) {
+    return code.replace('_nextBubbleNoFallback = "1"', "_nextBubbleNoFallback = undefined");
+  }
+
+  // The Next.js transpiled code contains something like `(0, _requestmeta.addRequestMeta)(req, "bubbleNoFallback", true);`
+  // and we want to update it to `(0, _requestmeta.addRequestMeta)(req, "bubbleNoFallback", false);`
+  return code.replace(/\((.*?.addRequestMeta\)\([^,(]*?,\s+"bubbleNoFallback"),\s+true\)/, "($1, false)");
```

Comment:
Why not just `.*?` instead of `[^,(]*?` ?
There should be no "(" and "," is already taken care of by "?"

Reply:
> Why not just .*? instead of [^,(]*?

it was like that originally but then I thought that we might want to make sure no extra parameter or unexpected things are present, but yeah I forgot that match is non greedy..., so yeah the `,` is unnecessary and the `(` is a bit pointless/unnecessary too, let me revert the change 👍 

> There should be no "("

yeah there shouldn't be one, and that's what I was checking, for example some code could have as the first parameter a function call and I wanted not to match that... but yeah I feel like that's quite an overkill here

Reply:
reverted 👍